docs/json/sonarr/cf/nf.json

```
@@ -13,7 +13,7 @@
       "negate": false,
       "required": true,
       "fields": {
-        "value": "\\b(nf|netflix)\\b"
+        "value": "\\b(nf|netflix(u)?(hd)?)\\b"
```

https://regex101.com/r/pWZokU/1
```suggestion
        "value": "\\b(nf|netflix(u?hd)?)\\b"
```